config/initializers/apartment.rb

```diff
@@ -9,15 +10,14 @@
   # Apartment Configuration
   #
   Apartment.configure do |config|
-
     # Add any models that you do not want to be multi-tenanted, but remain in the global (public) namespace.
     # A typical example would be a Customer or Tenant model that stores each Tenant's information.
-    config.excluded_models = %w{ Account AccountCrossSearch DomainName Endpoint User UserStat SolrEndpoint FcrepoEndpoint RedisEndpoint }
+    config.excluded_models = %w[Account AccountCrossSearch DomainName Endpoint User UserStat SolrEndpoint FcrepoEndpoint RedisEndpoint GoodJob::Execution GoodJob::Job GoodJob::Process]
```

Comment:
we probably need to only add GJ classes if GD is enabled.

```suggestion
    config.excluded_models = %w[Account AccountCrossSearch DomainName Endpoint User UserStat SolrEndpoint FcrepoEndpoint RedisEndpoint]
    config.exlucded_models += %w[GoodJob::Execution GoodJob::Job GoodJob::Process] if ENV.fetch('HYRAX_ACTIVE_JOB_QUEUE', 'sidekiq') == 'good_job'
```